test/infrastructure/docker/templates/cluster-template-development-mp.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cluster.x-k8s.io/v1beta1
+apiVersion: cluster.x-k8s.io/v1beta2
```

Comment:
Did you test these 3 templates manually? (+ the CCs)

Quickest way would be to apply all of them with tilt

Reply:
No, I didn't.

> Quickest way would be to apply all of them with tilt

I'd try it. Thanks!